modules/logging-loki-reliability-hardening.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-loki.adoc
+
+:_content-type: REFERENCE
+[id="logging-loki-reliability-hardening_{context}"]
+= Configuring Loki to tolerate node failure
+
+In logging 5.8 and newer versions, the Loki Operator supports setting pod anti-affinity rules to request that pods of the same component are scheduled on different available nodes in the cluster.
+
+include::snippets/about-pod-affinity.adoc[]
+
+[id="logging-loki-reliability-hardening-default-settings_{context}"]
+== Default Loki settings
+
+The Operator sets default, preferred `podAntiAffinity` rules for all Loki components, which includes the `compactor`, `distributor`, `gateway`, `indexGateway`, `ingester`, `querier`, `queryFrontend`, and `ruler` components.
+
+.Example default preferred rule settings for the ingester component
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
```

Comment:
I don't think we need to specify the default settings in the LokiStack CR. the podAntiAffinity rules should be applied  by default when a CR is created for all loki components. For e.g.
```
$ oc get statefulset lokistack-dev-ingester -o json | jq -r '.spec.template.spec.affinity'
{
  "nodeAffinity": {
    "requiredDuringSchedulingIgnoredDuringExecution": {
      "nodeSelectorTerms": [
        {
          "matchExpressions": [
            {
              "key": "kubernetes.io/os",
              "operator": "In",
              "values": [
                "linux"
              ]
            }
          ]
        }
      ]
    }
  },
  "podAntiAffinity": {
    "preferredDuringSchedulingIgnoredDuringExecution": [
      {
        "podAffinityTerm": {
          "labelSelector": {
            "matchLabels": {
              "app.kubernetes.io/component": "ingester",
              "app.kubernetes.io/instance": "lokistack-dev"
            }
          },
          "topologyKey": "kubernetes.io/hostname"
        },
        "weight": 100
      }
    ]
  }
}
```

Reply:
Here the CR created had below spec 
```
spec:
  managementState: Managed
  rules:
    enabled: true
    namespaceSelector:
      matchLabels:
        openshift.io/cluster-monitoring: 'true'
    selector:
      matchLabels:
        openshift.io/cluster-monitoring: 'true'
  size: 1x.demo
  storage:
    schemas:
      - effectiveDate: '2022-06-01'
        version: v12
    secret:
      name: test
      type: s3
  storageClassName: gp2-csi
  tenants:
    mode: openshift-logging
```

Reply:
Removed information about the default config, kept only information about how to override the default config